app/controllers/verify/phone_controller.rb

```diff
@@ -6,6 +6,8 @@ class PhoneController < ApplicationController
     before_action :confirm_step_allowed
 
     helper_method :idv_phone_form
+    helper_method :remaining_step_attempts
+    helper_method :step_name
```

Comment:
if you went for a view model, `step_name` could also be a method on each view model. :)